canonical/canonical.com
# Create new /documentation pages

## Done

- Updates /practice/documentation overview page
- Build the new pages /work-and-careers and /beyond-canonical
- Create a tab navigation to move between the different pages, based on https://charmhub.io/mongodb-k8s 
- Make JS reusable
- Add an image that was missing from the /beyond-canonical copydoc

## QA

[overview copydoc](https://docs.google.com/document/d/10tOsj5kKZRddQYYkb1PgxsES47Qic8XJc9xpRaYHd7s/edit#)
[work and careers copydoc](https://docs.google.com/document/d/1U-IkZOMzQlfV0zAAmM0PRVUBoZNBeOa-Qrso3i4rDVU/edit#)
[beyond canonical copydoc](https://docs.google.com/document/d/12VyKotmcjOD7zRGl_9vLcrKXZFvhAKJ1ZNnFAiWKWmk/edit#)

- Go to each age individually and compare against the relevant copydocs
- Go to https://canonical-com-776.demos.haus/documentation and use the tabs near the top of the page to navigate between the different pages. Check it takes you to the right page and highlights the correct tab.
- Check the side navigation works on each page. Upon clicking it should take you to the associated section and highlight it. Upon scrolling it should still highlight the section most visible.

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-1395
